Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9
+
+WORKDIR /vaccinewatcher
+COPY . . 
+
+# Install dependencies
+COPY requirements.txt .
+RUN pip3 install -r requirements.txt 
+
+# Run application
+CMD [ "python", "vaccinewatcher/watcher.py" ]
```

Comment:
This line should be `ENTRYPOINT` instead of `CMD` because this is a command-line script.